### PR TITLE
Add example for saving / loading models using IndexedDB to Iris demo

### PR DESCRIPTION
**Pull request by caisq**
* Remove the capability to load a converted Keras model from the local
  file system, a) to avoid confusion, b) this functionality is not
  highly useful anymore.
* Fix an issue in the widths of the vega-embed plots: they somehow
  started resizing. This PR makes then fixed, as they should be.

Towards: https://github.com/tensorflow/tfjs/issues/13

---
This change is [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-examples/87)

### CODE REVIEW

**Comment by davidsoergel** [comment]
:lgtm_strong:

---

Reviewed 5 of 7 files at r1, 2 of 2 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---

*[iris/loader.js, line 51 at r2](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-LDI7vG47TCZ4Flfe8sO:-LDI7vG47TCZ4Flfe8sP:bfjqgpl) ([raw file](https://github.com/tensorflow/tfjs-examples/blob/e8e0112ebd66a3179337300bf23931c1c47638dd/iris/loader.js#L51)):*
> ```
> client-side where
> ```
nit: client-side location where

---

*[iris/serve.sh, line 31 at r2](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-LDI8FN93wGy2oIwY7yq:-LDI8FN93wGy2oIwY7yr:b-9mw4yy) ([raw file](https://github.com/tensorflow/tfjs-examples/blob/e8e0112ebd66a3179337300bf23931c1c47638dd/iris/serve.sh#L31)):*
> ```Shell
> 
> # Ensure that http-server is available
> # yarn
> ```

Should this be commented out?

---


*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-:-LDIBEby2ynVog3juOy5:b-jbsf8z)*

File: iris/serve.sh [hunk 1]
```diff
@@ -28,14 +28,14 @@ NODE_ENV=development
 RESOURCE_PORT=1235
 
 # Ensure that http-server is available
-yarn
+# yarn
```
Comment (bileschi) [inline iris/serve.sh]: Is this comment meant to be maintained?  Does it make sense to keep it?

**Comment by bileschi** [comment]
:lgtm_strong:

---

Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-:-LDMo89KDA1SjELkc6l8:b-jbsf8z)*

**Comment by caisq** [comment]
Thanks for your reviews!

---

Review status: all files reviewed at latest revision, 3 unresolved discussions, all commit checks successful.

---

*[iris/loader.js, line 51 at r2](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-LDI7vG47TCZ4Flfe8sO:-LDNMXwrBx6NgRTR0jtj:b-896fix) ([raw file](https://github.com/tensorflow/tfjs-examples/blob/7981b165557081279e34ba91c7075e50a99bccc0/iris/loader.js#L51)):*
<details><summary><i>Previously, davidsoergel (David Soergel) wrote…</i></summary><blockquote>

> ```
> client-side where
> ```
nit: client-side location where
</blockquote></details>

Done.

---

*[iris/serve.sh, line 31 at r2](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-LDI8FN93wGy2oIwY7yq:-LDNMcz2CdlSRSE8nqOs:b-bvwhm5) ([raw file](https://github.com/tensorflow/tfjs-examples/blob/7981b165557081279e34ba91c7075e50a99bccc0/iris/serve.sh#L31)):*
<details><summary><i>Previously, davidsoergel (David Soergel) wrote…</i></summary><blockquote>

Should this be commented out?
</blockquote></details>

Restored.

---

*[iris/serve.sh, line 31 at r2](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-LD7Y0Z1zazSFMj7cX9V-r2-31:-LDNMeEW68BxWS7dgZ_r:bo42psd) ([raw file](https://github.com/tensorflow/tfjs-examples/blob/7981b165557081279e34ba91c7075e50a99bccc0/iris/serve.sh#L31)):*
<details><summary><i>Previously, bileschi (Stanley Bileschi) wrote…</i></summary><blockquote>

Is this comment meant to be maintained?  Does it make sense to keep it?
</blockquote></details>

Restored `yarn`.

---


*Comments from [Reviewable](https://reviewable.io/reviews/tensorflow/tfjs-examples/87#-:-LDNMv90-LAbaY_qHLn_:b7absbu)*

**Pull request merged**